launchlib/processors.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2016 Palantir Technologies, Inc.
```

Comment:
Perhaps the limit of my ability to review golang code ;-)
```suggestion
// Copyright 2023 Palantir Technologies, Inc.
```

Reply:
Updated... looks like the default godel license configuration handles the "year", so probably shouldn't have been [hardcoding 2016](https://github.com/palantir/go-java-launcher/blob/dd2531328daa91e4baa2eb85e9c43d7468cae7dc/godel/config/license-plugin.yml#L2).